grin/grin/live-variable/case_fun_side_effect.grin

```diff
@@ -0,0 +1,11 @@
+grinMain =
```

Comment:
Please remove these example `.grin` files. We should put them into the repo when we have some end-to-end test case.

Reply:
Done.